flow360/component/simulation/operating_condition.py

```diff
@@ -21,22 +25,21 @@
 from flow360.log import log
 
 # pylint: disable=no-member
-VelocityVectorType = Union[VelocityType.Vector, Tuple[pd.StrictStr, pd.StrictStr, pd.StrictStr]]
+VelocityVectorType = Union[Tuple[pd.StrictStr, pd.StrictStr, pd.StrictStr], VelocityType.Vector]
 
 
 class ThermalStateCache(Flow360BaseModel):
     """[INTERNAL] Cache for thermal state inputs"""
 
     # pylint: disable=no-member
-    constructor: Optional[str] = None
     altitude: Optional[LengthType.Positive] = None
     temperature_offset: Optional[TemperatureType] = None
     temperature: Optional[TemperatureType.Positive] = None
```

Comment:
We are not storing model fields in cache right? temperature and density should be deleted

Reply:
I think this depends on the front end design whether we need an extra layer when going from python client JSON to front end representation. Let's keep as is for now.